src/Experiment/tests/MyClassTests.cs

```diff
@@ -1,14 +1,68 @@
 using System;
```

Comment:
License header is needed here. Also, let's rename the file to what it is testing.